easybuild/easyconfigs/r/R/R-3.1.1-intel-2014b.eb

```diff
@@ -222,6 +222,7 @@ exts_list = [
     ('GenomicAlignments', '1.0.4', bioconductor_options),
     ('ShortRead', '1.22.0', bioconductor_options),
     ('graph', '1.42.0', bioconductor_options),
+    ('doMC', '1.3.3', bioconductor_options),
```

Comment:
we won't start updating old R easyconfigs; please flesh this out as a stand-alone easyconfig?